docs/www/configuration.md

```diff
@@ -622,6 +622,19 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can be restarted and troubleshot.
+  # Default: false
+  tune_ballast_file: false
+
+  # The path where the ballast file will be created.
+  # Default: "/var/lib/redpanda/data/ballast"
+  ballast_file_path: "/var/lib/redpanda/data/ballast"
+
+  # The ballast file size.
+  # Default: "1GiB"
+  ballast_file_size: "1GiB"
```

Comment:
why not default it to 0? , and remove the tuner boolean check 

can we also make the change in k8s operator to add bllast file of 1G

Reply:
I based this on the previous discussion: https://github.com/vectorizedio/redpanda/issues/322#issuecomment-749332437